pkg/controller/v1alpha1/inferencegraph/controller.go

```diff
@@ -249,6 +249,7 @@ func (r *InferenceGraphReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		}
 		hostname, err := routeReconciler.Reconcile(ctx, graph)
 		url.Host = hostname
+		url.Scheme = "https"
```

Comment:
Recently, I noticed that several users have been requesting support for the HTTP protocol. Instead of hardcoding this scheme, I think we need a more flexible approach to accommodate different requirements.

Reply:
I think we should create an RFE. I'll do it.